utils/src/main/kotlin/Utils.kt

```diff
@@ -288,6 +268,14 @@ fun normalizeVcsUrl(vcsUrl: String): String {
     return url
 }
 
+/**
+ * Resolve the Windows [executable] to its full name including the optional extension.
+ */
+fun resolveWindowsExecutable(executable: File): File? {
+    val extensions = Os.env["PATHEXT"]?.splitToSequence(File.pathSeparatorChar).orEmpty()
```

Comment:
If these functions are called multiple times during an ORT run, would it make sense to create a lazy property for `extensions`? Also for `paths` used by `getPathFromEnvironment()`?

Reply:
That is also something I though about, but in the end decided against it, for the following reasons:

- The operation is rather cheap.
- It's a bit unclear where to put such a lazy property for top-level functions. I wouldn't want them to be top-level, too; so probably in the `Os` or `Environment` class.
- I like the idea to this be a sequence, but as sequences can be only consumed once, I`d have to change that for a lazy property 😊 

That said, I'm still not totally against using lazy properties here, but I first would like to hear what you think about my argument 😁